src/lib.rs

```diff
@@ -60,6 +63,39 @@ pub fn inspect(config: &Config) -> Result<(), InspectError> {
 
     if config.plain {
         println!("{}", output);
+    }
+    else if config.unpretty.starts_with("flowgraph"){
+        if let Some(path) = &config.input {
+            let input_path = PathBuf::from(path);
+            let mut output_path = PathBuf::new();
+            output_path.push(input_path.file_name().unwrap());
+            output_path.set_extension("png");
+
+            // Create a temporary file to dump out the plain output
+            let tmp_file_path = tmpfile()?;
+            {
+                let mut file = File::create(&tmp_file_path)?;
+                {
+                    file.write_all(output.as_bytes())?;
+                }
+            }
+
+            // For now setup the correct `dot` arguments to write to a png
+            let output_str = output_path.to_str()
+                .ok_or(InspectError::Generic(String::from("Failed to convert output path to string.")))?;
```

Comment:
We could introduce a specific error for that. How about `InspectError::Flowgraph`?